src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/newinitiate/PostInitiateByIdControllerTest.java

```diff
@@ -438,6 +439,75 @@ void should_return_201_with_task_unassigned() throws Exception {
 
     }
 
+    @Test
```

Comment:
what about the scenario for a valid permission types with space?

Reply:
PermissionTypes are java enums (constants) with no spaces allowed. This story removes leading and trailing spaces of any permission types from DMNs.